WordPress/WordPressApi/WordPressComOAuthClient.swift

```diff
@@ -164,15 +161,14 @@ final class WordPressComOAuthResponseSerializer: AFJSONResponseSerializer
             else {
                 return responseObject
         }
-        /*
-         Possible errors:
-         - invalid_client: client_id is missing or wrong, it shouldn't happen
-         - unsupported_grant_type: client_id doesn't support password grants
-         - invalid_request: A required field is missing/malformed
-         - invalid_request: Authentication failed
-         - needs_2fa: Multifactor Authentication code is required
-         */
 
+        /// Possible errors:
```

Comment:
Instead of documenting the enum, we could document each value, so if any is removed or changed, the comment will most likely be updated too.